src/resources/utils/debounce.ts

```diff
@@ -7,7 +7,7 @@
  * @returns {Function} A new function that delays the invocation of the input function.
  *
  * @example
- * debounce(() => console.log('Hello, world!'), 500);
```

Comment:
had to change it for when I search for console.log it shows up and I think I have a console log in my code :p